rm/rm-server/src/main/java/org/ow2/proactive/resourcemanager/db/NodeSourceData.java

```diff
@@ -61,6 +62,11 @@
 
     private Client provider;
 
+    /**
+     * name of the variable --> value of the variable
+     */
+    private Map<String, Object> infrastructureVariables;
```

Comment:
maybe it's better to initialise this with an empty map to avoid possible nullpointer exceptions